packages/jest-each/package.json

```diff
@@ -19,6 +19,7 @@
   "license": "MIT",
   "dependencies": {
     "@jest/types": "^24.8.0",
+    "@types/node": "*",
```

Comment:
needs to be added to `tsconfig` as well

Reply:
Per your other comment, I'm going to remove this from package.json rather than adding it to tsconfig.